src/btree/bt_read.c

```diff
@@ -184,6 +184,300 @@ __bt_reconstruct_delta(WT_SESSION_IMPL *session, WT_REF *ref, WT_ITEM *delta)
     return (ret);
 }
 
+/*
+ * __bt_unpacked_delta_key_cmp --
+ *     Compare two unpacked deltas
+ */
+static int
+__bt_unpacked_delta_key_cmp(
+  WT_SESSION_IMPL *session, const WT_CELL_UNPACK_DELTA_INT *a, const WT_CELL_UNPACK_DELTA_INT *b)
+{
+    WT_DECL_RET;
+    WT_ITEM key_a, key_b;
```

Comment:
These local variables are redundant

Reply:
We do need 2 distinct items to be passed to the __wt_comapre function, have I missed something here?

Reply:
Can we just directly use a and b instead of having key_a, key_b

Reply:
I don't think so, __wt_compare() only accepts WT_ITEM.